src/FSharp.Data.Fred/JsonApi.fs

```diff
@@ -1,668 +1,356 @@
-        /// Of course, you can search for multiple words like 'money' and 'stock'. Remember to url encode spaces (e.g. 'money%20stock').
-        | FullText 
-        /// Performs a substring search on series IDs. Searching for 'ex' will find series containing 'ex' anywhere in a series ID.
-        /// '*' can be used to anchor searches and match 0 or more of any character. Searching for 'ex*' will find series containing 'ex' at the beginning of a series ID.
-        /// Searching for '*ex' will find series containing 'ex' at the end of a series ID. It's also possible to put an '*' in the middle of a string.
-        /// 'm*sl' finds any series starting with 'm' and ending with 'sl'.
-        | SeriesId
+type SearchType = 
+    | FullText 
+    | SeriesId
 
-    ///   <summary>
-    ///    Order results by values of the specified attribute. 
-    ///   </summary>
-    ///   <category index="2">Unions</category>
-    [<RequireQualifiedAccessAttribute>]
-    type SearchOrder = 
-        /// Order search by rank
-        | SearchRank
-        /// Order search by id numbering order
-        | SeriesIdOrder
-        /// Order search by title 
-        | Title
-        /// Order search by units
-        | Units
-        /// Order search by frequency
-        | Frequency
-        /// Order search by seasonal adjustment
-        | SeasonalAdjustment 
-        /// Order search by most recent real time start
-        | RealTimeStart 
-        /// Order search by most recent real time end
-        | RealTimeEnd
-        /// Order search by most recent updated series
-        | LastUpdated
-        /// Order search by most recent observation start date
-        | ObservationStart
-        /// Order search by most recent observation end date 
-        | ObservationEnd
-        /// Order search by most popular
-        | Popularity
-        /// Order search by most popular groups
-        | GroupPopularity
+[<RequireQualifiedAccessAttribute>]
+type SearchOrder = 
+    | SearchRank
+    | SeriesIdOrder
+    | Title
+    | Units
+    | Frequency
+    | SeasonalAdjustment 
+    | RealTimeStart 
+    | RealTimeEnd
+    | LastUpdated
+    | ObservationStart
+    | ObservationEnd
+    | Popularity
+    | GroupPopularity
 
-    ///   <summary>
-    ///    Sort results is ascending or descending observation date order.
-    ///   </summary>
-    ///   <category index="3">Unions</category>
-    [<RequireQualifiedAccessAttribute>]
-    type SortOrder =
-        /// Sort by ascending date order
-        | Ascending
-        /// Sort by descending date order
-        | Descending
+[<RequireQualifiedAccessAttribute>]
+type SortOrder =
+    | Ascending
+    | Descending
 
-    ///   <summary>
-    ///   A key that indicates a data value transformation.
-    ///   </summary>
-    ///   <category index="4">Unions</category>
-    [<RequireQualifiedAccessAttribute>]
-    type Units =
-        /// Observation(t), default.
```

Comment:
See FSharp.Formatting APIDocs where it shows how to do math. These should all be in latex math style so that they're more clear.